crates/bls-crypto/src/hash_to_curve/mod.rs

```diff
@@ -61,6 +77,21 @@ pub trait HashToCurve {
     ) -> Result<Self::Output, BLSError>;
 }
 
+impl<Alg: CrhAndXofHashToCurve> HashToCurve for Alg
+{
+    type Output = Alg::Output;
+
+    fn hash(
```

Comment:
What do you think about finding a different name for the `hash` function, either here or the `CrhAndXofHashToCurve`?
The current situation creates create an ambiguity when calling the function, which you've seen in the tests.

Reply:
Definitely can do this! I have a preference for keeping `HashToCurve` minimal, so using the name `hash`. 

Perhaps we should change the name of `hash` on `CrhAndXofHashToCurve` to be `crh_and_xof_hash`? Its kind of long, but I'm blanking on any better idea

Reply:
I'll noodle on it :)

Reply:
bump on the name here, should I just go with `crh_and_xof_hash`?